ppl/detail/EventLimit.tsx

```diff
@@ -0,0 +1,29 @@
+import {Caption} from "app/detail/Shared"
+import React, {memo, useCallback} from "react"
+import {useDispatch} from "react-redux"
+import Link from "src/js/components/common/Link"
+import {submitSearch} from "src/js/flows/submitSearch/mod"
+import {UID_CORRELATION_LIMIT} from "src/js/searches/programs"
+import SearchBar from "src/js/state/SearchBar"
+
+type Props = {
+  count: number
+  query: string
+}
+
+export default memo<Props>(function EventLimit({query, count}) {
+  const dispatch = useDispatch()
+  const onClick = useCallback(() => {
+    dispatch(SearchBar.clearSearchBar())
+    dispatch(SearchBar.changeSearchBarInput(query))
+    dispatch(submitSearch())
+  }, [query])
+
+  if (count < UID_CORRELATION_LIMIT) return null
+  return (
+    <Caption>
+      Limited to {UID_CORRELATION_LIMIT} events.
+      <Link onClick={onClick}>Query for All</Link>
+    </Caption>
+  )
+})
```

Comment:
If there are more than 100 events returned for any correlation, we get this link where they can query for all of the events without the `| head 100` tacked on the end.